react-components/src/architecture/base/commands/CustomBaseInputCommand.ts

```diff
@@ -51,7 +61,33 @@ export abstract class CustomBaseInputCommand extends RenderTargetCommand {
     this._contents = contents;
 
     const invokeResult = this.invoke();
-    this._contents = this._contents.map((content) => ({ type: content.type, content: '' }));
+    this._contents = this._contents.map((content) => {
+      if (content.type === 'text' || content.type === 'comment') {
+        return {
+          type: content.type,
+          content: ''
+        };
+      }
+      if (content.type === 'commentWithButtons') {
+        return {
+          type: content.type,
+          content: ''
+        };
+      }
+      if (content.type === 'customInput') {
+        return {
+          type: content.type,
+          content: undefined
+        };
+      }
+      if (content.type === 'submitButtons') {
+        return {
+          type: content.type,
+          content: undefined
+        };
+      }
+      return content;
+    });
```

Comment:
Could this be a "clearFieldContent" function?

Reply:
yes sure. Let's do it after release. Including some refactoring on this logic, simplifying it or even not using this approach